<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePatientDialog</class>
 <widget class="QDialog" name="CreatePatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>381</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Priorité :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditName">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditPostCode">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Durée de la consultation :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDateEdit" name="dateEditConsultationDay">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2018</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Téléphone : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditAddress">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEditTown">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Commentaire : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditSurname">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QSlider" name="horizontalSliderPriority">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QTimeEdit" name="timeEditDuration">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="minimumTime">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="time">
      <time>
       <hour>1</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Ville :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Jour de consultation :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Prénom :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Code postal :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Création d'un nouveau patient</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditPhone">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QTextEdit" name="textEditComment">
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreatePatientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreatePatientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
